utils/utils.ts

```diff
@@ -61,6 +61,25 @@ export function getEnvironmentRequiredVars() {
     ? process.env.MNEMONIC_POLK
     : " oh oh";
 
+  const clusterNodeA = process.env.CLUSTER_NODE_A
+    ? process.env.CLUSTER_NODE_A
+    : "ws://node_alice:9944";
+  const clusterNodeB = process.env.CLUSTER_NODE_B
+    ? process.env.CLUSTER_NODE_B
+    : "ws://node_bob:9944";
+  const clusterNodeC = process.env.CLUSTER_NODE_C
+    ? process.env.CLUSTER_NODE_C
+    : "ws://node_charlie:9944";
+  const clusterNodeD = process.env.CLUSTER_NODE_D
+    ? process.env.CLUSTER_NODE_D
+    : "ws://node_dave:9944";
+  const clusterNodeE = process.env.CLUSTER_NODE_E
+    ? process.env.CLUSTER_NODE_E
+    : "ws://node_eve:9944";
+  const clusterNodeF = process.env.CLUSTER_NODE_F
+    ? process.env.CLUSTER_NODE_F
+    : "ws://node_ferdie:9944";
+
```

Comment:
As we were talking in Slack, I would leave the clusters definition a bit more flexible. Accepting a list of clusters and build the test on that direction, instead of having a pre-defined list of clusters ( on this case from A to F)
But this can be done later I think.